### ISSUE TITLE
where is 'prometheus.yaml' ??

### ISSUE THREAD

**Issue by A-RaPhaellL** [greenstatic/bigbluebutton-exporter#34]
Hi,
In this guide,

https://bigbluebutton-exporter.greenstatic.dev/installation/bigbluebutton_exporter/#docker-installation-recommended

In the eighth step says: "Add the exporter to your Prometheus configuration."
but question is where is it(prometheus.yaml) ???

**Comment by greenstatic** [comment]
The installation instructions for _BigBlueButton exporter_ assume you have a working Prometheus and Grafana installation, since BigBlueButton exporter is an extension that allows a BigBlueButton server to be "plugged in" your Grafana/Prometheus monitoring infrastructure.

Since many people don't have existing monitoring infrastructure there is an alternative installation targeting administrators who only have a single BigBlueButton server - [All in one monitoring stack](https://bigbluebutton-exporter.greenstatic.dev/installation/all_in_one_monitoring_stack/). This is not a scalable solution if you have multiple BigBlueButton servers because each BBB server would have a Prometheus/Grafana instance which is a bit of a nonsense.

If you have multiple BBB servers I suggest you follow Prometheus and Grafana installation instructions on how to setup "proper" monitoring infrastructure.

**Comment by A-RaPhaellL** [comment]
Hi,
Thanks for your guidance
We fix this problem by install 'Prometheus' and config it to 'Grafana'.
but now we have a new problem.

All of our panels show 'NO DATA'.

**Comment by greenstatic** [comment]
First check if your Prometheus instance is correctly configured and scraping the exporter metrics - https://bigbluebutton-exporter.greenstatic.dev/debugging/#checking-if-prometheus-is-scraping-correctly

If Prometheus is correctly scraping then the issue is between Prometheus <-> Grafana. Check that you have configured your data source correctly in the Grafana settings.

**Comment by A-RaPhaellL** [comment]
I run this command:
`ssh -NL 9090:127.0.0.1:9090 server.example.com`

and get this error:
`ssh: connect to host server.example.com port 22: Connection refused`

could you please say me what is the problem ?

**Comment by greenstatic** [comment]
You are connecting via SSH to your prometheus server. Are you sure you are connecting to it correctly - prometheus server ssh running on port 22, prometheus running on port 9090 and the correct host (server.example.com)?

**Comment by greenstatic** [comment]
Have you managed to debug your issue?

**Comment by A-RaPhaellL** [comment]
Thank you for your response and follow-up to solve my problem.
I think I found my problem.

When I check this link after install 'Exporter'.
`'example.com/metrics'`
It's showing me the '404 not found' page.

So I went and checked the 'docker'.
`docker ps -a`

And I saw my 'docker' restarting again on again.
You have any idea to help me?

**Comment by greenstatic** [comment]
FYI: The `-a` flag displays besides running/restarting containers, stopped containers. If your container keeps on restarting the flag isn't necessary.

Run `docker logs --tail 30 <container_name>`, the container name can be found from the `docker ps` output under the _NAMES_ column.  The last 30 lines should suffice to narrow down the issue (if you do not include tail option docker will output all the logs of your restarting container which is, well, a lot :) ).

**Comment by A-RaPhaellL** [comment]
Thanks again for your help.
My problem was solved.
I take a log and realized that 'docker' could not be connected to the 'recording' folder.
I found out that I don't have any record and maybe no folder has been created for it.
I make a record and tried the 'docker' again and everything worked fine.
I continue, I hope there will be no problem again.

**Comment by A-RaPhaellL** [comment]
After that, I had a new problem that I solved.
I realized that my 'bbb_node_exporter' could not be connected and show 'Unauthorized' error.

So I went and check the steps again, which was in Linz below.
https://bigbluebutton-exporter.greenstatic.dev/installation/node_exporter/

I realize that the fifth step has a problem.
No information about 'user' and 'pass' given to 'Prometheus'.
The correct file format should look like this:
```
  - job_name: 'bbb_node_exporter'
    metrics_path: '/node_exporter/metrics'
    params:
      format: [prometheus]
    honor_labels: true
    scheme: https
    basic_auth:
      username: "<HTTP BASIC AUTH USERNAME>"
      password: "<HTTP BASIC AUTH PASSWORD>"
    static_configs:
    - targets: ['example.com']
```

After fix that, the site information, came up in the 'Monitoring'.

Finally, with your help, I was able to do it.
Thank you for your time and guidance.
